src/components/dashboard/Hub/index.tsx

```diff
@@ -36,26 +36,18 @@ function HubRouter() {
   const { token } = useSessionDataState()
   const [hubCards, setHubCards] = useState(hubSteps)
   const { isEligible } = useEligibility()
-  const [error, setError] = useState()
-  const [isLoading, setIsLoading] = useState(false)
   const { push } = useHistory()
-  const { t } = useTranslation()
   const [userInfo, setUserInfo] = useState<LoggedInUserData | undefined>()
   const { checkpoint } = userInfo?.clientData || {}
-
+  const [showWelcome, setShowWelcome] = useState(!token)
   useEffect(() => {
     const getInfo = async () => {
       if (token) {
         try {
-          setIsLoading(true)
           const userInfoResponse = (await UserService.getUserInfo(token)) as any
-
           setUserInfo(userInfoResponse?.data)
         } catch (e) {
           console.error(e)
-          setError(e?.message)
```

Comment:
Is this call only during fetching initial data?
or also fetched each time data is saved into clientData?

- if its only on fetching - i think is important to show the error to the user about the loading spinner and if something fails; specially if they have slow connectivity on thier phones depending on the area.